.github/workflows/eamxx_default_files.yml

```diff
@@ -0,0 +1,72 @@
+name: inputdata
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '00 00 * * *'
+  workflow_dispatch:
+
+jobs:
+  scream-defaults:
+    runs-on: ubuntu-latest
+    outputs:
+      event_name: ${{ github.event_name }}
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          submodules: false
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4.7.0
+        with:
+          python-version: "3.11"
+      - name: Run unit tests
+        working-directory: components/eamxx/cime_config/
+        run: |
+          python -m unittest tests/eamxx_default_files.py -v
+
+  notify-scream-defaults:
+    needs: scream-defaults
+    if: failure() && needs.scream-defaults.outputs.event_name == 'pull_request'
```

Comment:
Where does "failure()" come from?

Reply:
Also, I think we should run this on master, not on the PR. The PR already has merge blocked, since the checks don't pass. Also, opening an issue b/c a PR branch is missing data seems wrong. OTOH, it is appropriate if master is missing data on the server.

Reply:
Ok. I was puzzled by the == instead of != typo. Also, I didn't know where `failure()` is defined. I guess it's a built-in function of the action framework.